code/src/components/Story.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+
+export const Story = ({ fairyName, color, horn, loveAct, name }) => {
+  return (
+    <>
+      <container className="story-page">
+        <p>
+          <span className="big-letter-span">O</span>nce upon a time, in a magical land far, far away, lived a beautiful,
```

Comment:
There's a pseudo-element you could have tried here, p::first-letter, to style the O. In general the text part of this component is a bit messy to look at. An alternative approach could have been to remove the class names, use a couple of divs and  nth-of-type to target specific spans. Then you would have needed to count though and it would have been way less easy to come back and make a color change, so I do see the point of this too. It's easy to know what each class is referring to with your good classnames.